packages/puppeteer-core/src/api/Page.ts

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import net from 'net';
```

Comment:
is it a node module? we cannot use node modules in Puppeteer because it has to run in the browser.